app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with model: @task, class: 'task-form' do |f| %>
+
+  <div>
+    <%= f.label :name  %>
+    <%= f.text_field :name %> </br>
+
+    <%= f.label :description %>
+    <%= f.text_field :description %> </br>
+
+    <%= f.label :completed_at %>
```

Comment:
Note that because `complete_at` is entered when a task is created, a new task is already marked as complete and the button for `mark_incomplete` appears. This creates a bit of confusion for the user. Consider a design that doesn't have the user make an entry `completed_at` in the form.